java/code/src/com/redhat/rhn/domain/user/UserFactory.java

```diff
@@ -703,30 +704,26 @@ public void setUserServerPreferenceValue(User user,
         return query.list();
     }
 
-    /**
-     * Return a list of all users.
-     *
-     * @return list of users.
-     */
-    public List<User> findAllUsers() {
-        CriteriaBuilder builder = getSession().getCriteriaBuilder();
-        CriteriaQuery<UserImpl> criteria = builder.createQuery(UserImpl.class);
-        criteria.from(UserImpl.class);
-        return getSession().createQuery(criteria).getResultList().stream()
-                .<User>map(Function.identity()).collect(Collectors.toList());
-    }
-
     /**
      * Return a list of all User's who are in the given org.
```

Comment:
Return a list of all users in a given Org, or all users in all Orgs.

Reply:
Will be fixed by https://github.com/uyuni-project/uyuni/pull/59#discussion_r216752957